src/agent/multitype_queue/src/sqlitestorage.cpp

```diff
@@ -225,7 +262,7 @@ int SQLiteStorage::Remove(int id, const std::string& tableName, const std::strin
     }
     else
     {
-        constexpr std::string_view DELETE_QUERY {"DELETE FROM {} WHERE module LIKE \"{}\" AND rowid = ?;"};
+        constexpr std::string_view DELETE_QUERY {"DELETE FROM {} WHERE module_name LIKE \"{}\" AND rowid = ?;"};
```

Comment:
Same here.

Reply:
In this first stage, because of the way events are being sent, it is not necessary, but it should be done in the future.